### PR TITLE
#741 add checks if intersections active

### LINKED ISSUE #741: Missing check on whether specified intersections are active for the time they are pulled.
`get_intersection_info` can pull info for user specified intersections.

https://github.com/CityofToronto/bdit_data-sources/blob/6466d4fc93666b3bbd698a08482e4f4404ec1f33/volumes/miovision/api/intersection_tmc.py#L415-L431

But the check after assumes the function is filtering out intersections which aren't active for the requested date range, which isn't true.

https://github.com/CityofToronto/bdit_data-sources/blob/6466d4fc93666b3bbd698a08482e4f4404ec1f33/volumes/miovision/api/intersection_tmc.py#L440-L444

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
A very small PR which adds some checks to Miovision `intersections_tmc.py` to check and exit if:
  - no intersections found
  - no intersections active between start and end times specified

## Issue(s) this solves:
- Closes #741 

## What needs to be done by a sysadmin after this PR is merged
Nothing

### CODE REVIEW

**Comment by chmnata** [comment]
Looks good, has this been tested?

**Comment by gabrielwol** [comment]
I was able to test this one using the output from `get_intersection_info`! Sorry I didn't document it - if you want I can rerun those tests.

File: volumes/miovision/api/intersection_tmc.py [hunk 1]
```diff
@@ -450,16 +450,27 @@ def pull_data(conn, start_time, end_time, intersection, path, pull, key, dupes):
...
+    check_active = []
+    for c_start_t in daterange(start_time, end_time, time_delta):
+        active_intersections = [x.is_active(c_start_t) for x in intersections]
+        check_active.extend(active_intersections)
+
+    if not(any(check_active)):
+        if len(intersections) == 0:
+            logger.critical('No intersections found.')
+        elif user_def_intersection:
+            logger.critical('None of the specified intersections are active'
```
[the remark this answers is not in the document]
Comment (gabrielwol) [reply volumes/miovision/api/intersection_tmc.py]: Done!

**Review by chmnata — APPROVED** [review]

**Comment by chmnata** [comment]
pulled to data_script! deleting the branch

**Pull request merged**